tests/test_psf_trampoline.py

```diff
@@ -158,6 +197,17 @@ def testClone(self):
                     p.computeImage()
                 )
 
+    def testPersistence(self):
+        for pgp in self.pgps:
+            assert cppLib.isPersistable(pgp)
+            im = ExposureF(10, 10)
+            im.setPsf(pgp)
+            assert im.getPsf() == pgp
+            with lsst.utils.tests.getTempFilePath(".fits") as tmpFile:
+                im.writeFits(tmpFile)
+                newIm = ExposureF(tmpFile)
+                assert newIm.getPsf() == im.getPsf()
```

Comment:
We prefer to use the `unittest` module's assert methods (e.g. `self.assertEqual`) in our test code (at least for consistency; I also think it provides better printing of failures in at least some contexts).